src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetCreateOrUpdateMethod.cs

```diff
@@ -166,5 +322,83 @@ public override void ExecuteCmdlet()
             ValueFromPipeline = true)]
         [AllowNull]
         public PSVirtualMachineScaleSet VirtualMachineScaleSet { get; set; }
+
+        // SimpleParameterSet
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        [PSArgumentCompleter(
+            "CentOS",
+            "CoreOS",
+            "Debian",
+            "openSUSE-Leap",
+            "RHEL",
+            "SLES",
+            "UbuntuLTS",
+            "Win2016Datacenter",
+            "Win2012R2Datacenter",
+            "Win2012Datacenter",
+            "Win2008R2SP1")]
+        public string ImageName { get; set; } = "Win2016Datacenter";
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = true)]
+        public PSCredential Credential { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public int InstanceCount { get; set; } = 2;
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public string VirtualNetworkName { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public string SubnetName { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public string PublicIpAddressName { get; set; }
+        
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public string DomainNameLabel { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public string SecurityGroupName { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public string LoadBalancerName { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public int[] BackendPort { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        [LocationCompleter("Microsoft.Compute/virtualMachines")]
```

Comment:
SHouldn't this be virtualMachineScaleSets  ?

Reply:
good catch